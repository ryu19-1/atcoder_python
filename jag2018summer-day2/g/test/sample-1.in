4
1 7 3 5 5 7
1 4 1 2 5 4
6 1 7 1 7 6
11 3 11 4 8 5
