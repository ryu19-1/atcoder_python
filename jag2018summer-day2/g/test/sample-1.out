3 6
2 3
-1 -1
10 4
